UIComponents/Sources/UIComponents/ViewControllers/BaseVC.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+import Foundation
+import Combine
+
+public class BaseVC: UIViewController {
```

Comment:
Can we use BaseViewController instead of BaseVC? From my point of view, it's a bit weird)

Reply:
sure:)